tests/middleware/tests.py

```diff
@@ -389,7 +389,7 @@ class MyCommonMiddleware(CommonMiddleware):
 
 @override_settings(
     IGNORABLE_404_URLS=[re.compile(r"foo")],
-    MANAGERS=[("PHD", "PHB@dilbert.com")],
+    MANAGERS=["manager@example.com"],
```

Comment:
(dilbert.com is a real domain owned by someone other than DSF. As a general rule, we should avoid using those in tests and docs.)